synapse/models/economic.py

```diff
@@ -105,6 +130,12 @@ def getModelDefs(self):
 
                     ('pin', ('econ:pay:pin', {}), {
                         'doc': 'The Personal Identification Number on the card.'}),
```

Comment:
should this include "(PIN)" for consistency (with SWIFT, BIC, RTN, etc) or is PIN common enough to ignore?

Reply:
Not related a change in the PR but i'll tweak :+1: